main.tf

```diff
@@ -22,6 +22,7 @@ resource "azapi_resource" "this" {
   create_headers            = var.enable_telemetry ? { "User-Agent" : local.avm_azapi_header } : null
   delete_headers            = var.enable_telemetry ? { "User-Agent" : local.avm_azapi_header } : null
   read_headers              = var.enable_telemetry ? { "User-Agent" : local.avm_azapi_header } : null
+  response_export_values    = ["*"]
```

Comment:
Please can you be specific about the attributes you want to export? This saves clutter in the state file. Thanks!